circle.yml

```diff
@@ -9,7 +9,6 @@ dependencies:
   pre:
     - gem install fcrepo_wrapper
     - gem install solr_wrapper
-    - npm install phantomjs@1.9.8 
```

Comment:
Im not too sure. The tests seemed to pass without it. Im curious to see if they will keep passing. Ill add it back in, but if we can eventually take it out, that would be awesome.